src/deadline/client/cli/_groups/job_group.py

```diff
@@ -151,7 +151,7 @@ def job_get(**args):
     "--mark-as",
     type=click.Choice(["CANCELED", "FAILED", "SUCCEEDED"], case_sensitive=False),
     default="CANCELED",
-    help="The run status to mark the job as.",
+    help="The task run status to mark the job as.",
```

Comment:
I think we can still do better here... Is "task run status" exposed much to the users? I guess jobs that were not completed, canceled, or failed, will be set to the value they put in here. Maybe something like:

"The task status to change all active tasks to."

I wanted to avoid saying an actual status otherwise we would have list out all relevant status' which I also wanted to avoid. Open to ideas here though.

Reply:
How about, "The status to apply to all active tasks in the job."